src/datasets/builder.py

```diff
@@ -105,11 +105,11 @@ class BuilderConfig:
 
     def __post_init__(self):
         # The config name is used to name the cache directory.
-        invalid_windows_characters = r"<>:/\|?*"
-        for invalid_char in invalid_windows_characters:
+        invalid_windows_characters_in_path = r"<>:/\|?*"
```

Comment:
Maybe worth defining it as a constant at the module level? It is used again afterwards.

Reply:
I moved `INVALID_WINDOWS_CHARACTERS_IN_PATH` to `naming.py` :)